pkgs/by-name/po/portmaster/package.nix

```
@@ -0,0 +1,291 @@
+{
+  lib,
+  buildGoModule,
+  buildNpmPackage,
+  fetchFromGitHub,
+  pkg-config,
+  makeWrapper,
+  nodejs,
+  glib,
+  gtk3,
+  cairo,
+  pango,
+  gdk-pixbuf,
+  atk,
+  webkitgtk_4_1,
+  libsoup_3,
+  openssl,
+  curl,
+  systemdLibs,
+  iptables,
+  iproute2,
+  xorg,
+  alsa-lib,
+  nss,
+  nspr,
+  at-spi2-atk,
+  cups,
+  dbus,
+  expat,
+  fontconfig,
+  freetype,
+  zlib,
+  libayatana-appindicator,
+  coreutils,
+  zip,
+  rustPlatform,
+  cargo-tauri,
+  wrapGAppsHook4,
+  librsvg,
+  makeDesktopItem,
+  copyDesktopItems,
+  autoPatchelfHook,
+}:
+
+let
+  portmasterUI = buildNpmPackage rec {
```

```suggestion
  portmasterUI = buildNpmPackage (finalAttrs: {
```

Then, it is better to switch to finalAttrs.
Same for other places.